bots/zephyr_mirror_backend.py

```diff
@@ -206,7 +206,7 @@ def update_subscriptions():
         f = open(options.stream_file_path, "r")
         public_streams = simplejson.loads(f.read())
         f.close()
-    except:
+    except IOError:
         logger.exception("Error reading public streams:")
         return
```

Comment:
I think this should be an `except Exception`, since it's clearly intended to be a catch-all (to produce accurate logging), not handle a specific single error (and I'm pretty sure there are other exceptions that can happen here).